Gemfile

```diff
@@ -1,7 +1,9 @@
 source "https://rubygems.org"
 
 gem 'danger'
+gem 'rb-readline'
```

Comment:
This was added as a solution to this issue I ran into: https://github.com/fastlane/fastlane/issues/21794.